web/src/components/ThirdPartyProviderContainer/ThirdPartyProviderContainer.js

```diff
@@ -0,0 +1,26 @@
+import ThirdPartyProviderButton from 'src/components/ThirdPartyProviderButton'
+import Spinner from '../Spinner'
+
+export default ({ providers, onProviderClick, loading, ...rest }) => {
+  if (loading) {
+    return (
+      <div className="flex justify-center items-center">
```

Comment:
Instead of a spinner component, could you use:

https://tailwindcss.com/docs/animation#spin

?